geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/ingest/AbstractIngest.scala

```diff
@@ -231,8 +231,9 @@ object AbstractIngest {
   }
 
   private def buildString(c: Char, length: Int) = {
-    val sb = new StringBuilder(length)
-    (0 until length).foreach(_ => sb.append(c))
+    val slength = if (length < 0) 0 else length
```

Comment:
You know how it goes...  The smallest PRs get the most comments/line.  Couldn't we just reduce the object creation by saying something like:

```scala
if (length < 0) return ""
val sb = new StringBuilder(length)
(0 until length).foreach(_ => sb.append(c))
sb.toString()
```